packages/language-server/test/snapshots/xml-view-diagnostics/invalid-type-of-aggregation/input.xml

```diff
@@ -0,0 +1,16 @@
+<mvc:View xmlns:uxap="sap.uxap" xmlns:m="sap.m"
+    xmlns:mvc="sap.ui.core.mvc" 
+    xmlns="sap.ui.commons">
+    <m:Page>
+        <m:footer>
+            <🢂🢂Toolb🢀ar🢀>
```

Comment:
why are there double ranges here? I think you should pick
an example that is **only** wrong type and not **also** deprecated.
(see the error messages below)